apps/full-site-editing/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@automattic/full-site-editing",
+	"version": "1.0.0-alpha.0",
+	"description": "Plugin and theme enabling full site editing with the block editor.",
+	"sideEffects": true,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/full-site-editing"
+	},
+	"private": true,
+	"author": "Automattic, Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"scripts": {
+		"plugin": "webpack --folder='plugin' --env.WP='true'",
+		"dev:plugin": "npx lerna run plugin",
+		"build:plugin": "NODE_ENV=production npx lerna run plugin",
+		"theme": "webpack --folder='theme' --env.WP='true'",
+		"dev:theme": "npx lerna run theme",
+		"build:theme": "NODE_ENV=production npx lerna run theme"
+	},
+	"devDependencies": {
```

Comment:
Note this comment about devDependencies:

https://github.com/Automattic/wp-calypso/pull/32294/files#r276148646

It's mentioned in passing also in https://github.com/Automattic/wp-calypso/blob/master/docs/monorepo.md